### ISSUE TITLE
Clean html, css and javascript code

### ISSUE THREAD

**Issue by ManidharVutla** [ManidharVutla/Stock-Info#2]
Website code is from a downloaded online template. Code cleanup and files removal that doesn't impact the product.

**Comment by Samridhi015** [comment]
Hey! I can help you with this.

**Comment by ManidharVutla** [comment]
> Hey! I can help you with this.

Yeah, please go ahead. If you need assistance for setup, let me know. You just need to install the python packages required and you are all set to go. Run main.py
